### PR TITLE
Feat/user settings

### PR DESCRIPTION
**Pull request by josesrodriguez610**

### CODE REVIEW

File: src/components/Settings/CardAddMembers.jsx [hunk 1]
```diff
@@ -17,23 +18,24 @@ function CardAddMembers({
...
-    input: propTypes.string.isRequired,
-    myLeague: propTypes.string.isRequired
+    input: propTypes.string.isRequired
   };
 
+  const settings = useSelector(selectSettings);
+
   return (
-    <Typography>
+    <Typography align='center' style={{ paddingLeft: '20px' }}>
```
Comment (jstrzesz) [inline src/components/Settings/CardAddMembers.jsx]: no in-line styling. For material-ui, use the makeStyles or useStyles syntax.

File: src/components/Settings/CardSettingsL.jsx [hunk 2]
```diff
@@ -1,66 +1,49 @@
...
-      })
-      .then(() => axios.get(`/league/${league}/${user?.id}`))
-      .then((response) => setMyLeague(response.data));
+    handleFormSubmit: propTypes.func.isRequired,
+    handleChange: propTypes.func.isRequired
   };
 
   return (
-    <Typography>
+    <Typography align='center' style={{ paddingLeft: '50px' }}>
```
Comment (jstrzesz) [inline src/components/Settings/CardSettingsL.jsx]: no-line styling

File: src/components/Settings/CardUserSettings.jsx [hunk 3]
```diff
@@ -32,13 +33,16 @@ function CardUserSettings() {
   ];
 
   return (
-    <Typography>
+    <Typography
+      align='center'
+      style={{ paddingRight: '20px' }}
```
Comment (jstrzesz) [inline src/components/Settings/CardUserSettings.jsx]: see other comments on inline styling

Comment (josesrodriguez610) [reply src/components/Settings/CardUserSettings.jsx]: fixed

**Review by ChristopherStumpe — APPROVED** [review]

**Review by scottanderson1130 — APPROVED** [review]

**Pull request merged**